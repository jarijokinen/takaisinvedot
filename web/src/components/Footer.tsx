import * as Styled from './Footer.styled';

const Footer: React.FC = () => (
  <Styled.Footer>
    <Styled.Line>Palvelun toteutus: Jari Jokinen.</Styled.Line>{' '}
    <Styled.Line>
      Lähdekoodi löytyy{' '}
      <Styled.Link href="https://github.com/jarijokinen/takaisinvedot">
        GitHubista
      </Styled.Link>
      .
    </Styled.Line>
  </Styled.Footer>
);

export default Footer;
