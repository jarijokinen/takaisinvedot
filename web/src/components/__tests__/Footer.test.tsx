import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
