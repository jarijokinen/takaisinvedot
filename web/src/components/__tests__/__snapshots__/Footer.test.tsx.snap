// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<DocumentFragment>
  <header
    class="sc-AxjAm cyrofx"
  >
    <span
      class="sc-AxirZ feYKcc"
    >
      Palvelun toteutus: Jari Jokinen.
    </span>
     
    <span
      class="sc-AxirZ feYKcc"
    >
      Lähdekoodi löytyy 
      <a
        class="sc-AxiKw bOTXee"
        href="https://github.com/jarijokinen/takaisinvedot"
      >
        GitHubista
      </a>
      .
    </span>
  </header>
</DocumentFragment>
`;
